Broken elkjs build

Trying to build the current master I get the following error 
```
Compiling module org.eclipse.elk.js.ElkJs
   Ignored 25 units with compilation errors in first pass.
Compile with -strict or with -logLevel set to TRACE or DEBUG to see all errors.
   [ERROR] Errors in 'file:/workspace/elkjs/build/src/elk/org/eclipse/elk/alg/layered/DotDebugUtil.java'
      [ERROR] Line 53: No source code is available for type java.io.StringWriter; did you forget to inherit a required module?
   Tracing compile failure path for type 'org.eclipse.elk.alg.layered.DotDebugUtil'
      [ERROR] Errors in 'file:/workspace/elkjs/build/src/elk/org/eclipse/elk/alg/layered/DotDebugUtil.java'
         [ERROR] Line 53: No source code is available for type java.io.StringWriter; did you forget to inherit a required module?
   [ERROR] Hint: Your source appears not to live underneath a subpackage called 'client'; no problem, but you'll need to use the <source> directive in your module to make it accessible
   [ERROR] Errors in 'file:/workspace/elkjs/build/src/elk/org/eclipse/elk/alg/layered/JsonDebugUtil.java'
      [ERROR] Line 56: No source code is available for type java.io.StringWriter; did you forget to inherit a required module?
   Tracing compile failure path for type 'org.eclipse.elk.alg.layered.JsonDebugUtil'
      [ERROR] Errors in 'file:/workspace/elkjs/build/src/elk/org/eclipse/elk/alg/layered/JsonDebugUtil.java'
```

Have there been recent changes to the two mentioned util classes?

Also, I'm wondering if the travis build is broken. After all, the last time this happened you were notified via mail. I'll double check.

--- a/plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/intermediate/ConstraintsPostprocessor.java
+++ b/plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/intermediate/ConstraintsPostprocessor.java
@@ -29,9 +29,11 @@ public final class ConstraintsPostprocessor implements ILayoutProcessor<LGraph> 
     public void process(final LGraph graph, final IElkProgressMonitor progressMonitor) {
 
         progressMonitor.begin("Constraints Postprocessor", 1);
+        // elkjs-exclude-start
         if (progressMonitor.isLoggingEnabled()) {
             DebugUtil.logDebugGraph(progressMonitor, graph, 0, "ConstraintPostProcessor");
         }
+        // elkjs-exclude-end
         // Uses for each loops since they should be more efficient because LinkedLists are accessed.
         // It still creates counter variables so that no getter calls on layer and node are needed.
         int layerIndex = 0;
@@ -49,9 +51,11 @@ public final class ConstraintsPostprocessor implements ILayoutProcessor<LGraph> 
 
             layerIndex++;
         }
+        // elkjs-exclude-start
         if (progressMonitor.isLoggingEnabled()) {
             DebugUtil.logDebugGraph(progressMonitor, graph, 0, "ConstraintPostProcessor");
         }
+        // elkjs-exclude-end
         progressMonitor.done();
 
     }

